ext/doc/Cassandra/Float.php

```diff
@@ -21,107 +21,118 @@
 /**
  * A PHP representation of the CQL `float` datatype
  */
-final class Float implements Value, Numeric
-{
-    /**
-     * Minimum possible Float value
-     *
-     * @return Float minimum value
-     */
-    public static function min() {}
+final class Float_ implements Value, Numeric {
```

Comment:
where does this class name change comes from ?

Reply:
[PSR-4](https://www.php-fig.org/psr/psr-4/) recommends having file name matches class name. Shouldn't this file be renamed to `ext/doc/Cassandra/Float_.php` ?